src/org/zaproxy/zap/extension/selenium/ZapAddOn.xml

```diff
@@ -1,6 +1,6 @@
 <zapaddon>
 	<name>Selenium</name>
-	<version>11</version>
+	<version>12</version>
 	<semver>2.0.0</semver><!-- This version should be kept in sync with the one of the extension. -->
```

Comment:
Should be updated to 3.0.0? `JxBrowserProvider is failing because of binary incompatibilities.

Reply:
Discussed this on irc and agree to stick with 2.0.0, will submit a PR for jxbrowser now...